### DISCUSSION TITLE
I'm trying to install it but I get this

### DISCUSSION THREAD

**Discussion by mazenzoor** [spatie/laravel-export#78] [Q&A]
Problem 1
    - spatie/crawler[4.7.0, ..., v4.x-dev] require guzzlehttp/psr7 ^1.4 -> found guzzlehttp/psr7[1.4.0, ..., 1.x-dev] but the package is fixed to 2.1.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - spatie/laravel-export 0.3.9 requires spatie/crawler ^4.7 -> satisfiable by spatie/crawler[4.7.0, ..., v4.x-dev].
    - Root composer.json requires spatie/laravel-export ^0.3.9 -> satisfiable by spatie/laravel-export[0.3.9].

**Comment by hridoymahmud71** [comment]
I am having a similar issue , the guzzle requirement for this package is old and I cannot downgrade guzzle due to other dependencies.
